exonum-java-binding-core/rust/integration_tests/tests/utils_errors.rs

```diff
@@ -146,6 +148,42 @@ fn get_and_clear_java_exception_if_exception_occurred() {
         .unwrap();
 }
 
+// FIXME blocked by https://jira.bf.local/browse/ECR-1998
+// Currently (jni 0.11) there is no way to throw exception from native code
+// without message argument.
```

Comment:
Isn't it possible to 
1. Create an exception using no-arg constructor using env.new_object?
2. Throw it with that method?